__tests__/app.test.js

```diff
@@ -182,3 +183,52 @@ describe("PATCH /api/articles/:article_id", () => {
       });
   });
 });
+describe("GET /api/articles", () => {
+  test("200: returns an articles object with an array of articles with correct keys", () => {
+    return request(app)
+      .get("/api/articles")
+      .expect(200)
+      .then(({ body }) => {
+        expect(body).toHaveProperty("articles");
```

Comment:
Could argue that this assertion to check 'articles' property exists is redundant when you're then using assertions below that require 'articles' to exist for them to work